tests/test_tokens_prices.py

```diff
@@ -0,0 +1,65 @@
+import setup
+import pytest
+import timeit
+import time
+
+from datetime import datetime
+from sqs_service import *
+from coingecko_service import *
+from conftest import SERVICE_MAP
+from conftest import SERVICE_COINGECKO
+from constants import *
+
+class TestTokensPrices:
+
+    # NUM_TOKENS_DEFAULT low liquidity tokens
+    @pytest.mark.parametrize("token",setup.choose_tokens_liq_range(NUM_TOKENS_DEFAULT, MIN_LIQ_FILTER_DEFAULT, MAX_VAL_LOW_LIQ_FILTER_DEFAULT))
+    def test_low_liq_token_prices(self, environment_url, token):
+        self.run_coingecko_comparison_test(environment_url, token, HIGH_PRICE_DIFF)
+
+    # NUM_TOKENS_DEFAULT low volume tokens
+    @pytest.mark.parametrize("token",setup.choose_tokens_volume_range(NUM_TOKENS_DEFAULT, MIN_VOL_FILTER_DEFAULT, MAX_VAL_LOW_VOL_FILTER_DEFAULT))
+    def test_low_volume_token_prices(self, environment_url, token):
+        self.run_coingecko_comparison_test(environment_url, token, HIGH_PRICE_DIFF)
+
+    # NUM_TOKENS_DEFAULT mid volume tokens
+    @pytest.mark.parametrize("token",setup.choose_tokens_volume_range(NUM_TOKENS_DEFAULT, MIN_VOL_FILTER_DEFAULT, MAX_VAL_MID_VOL_FILTER_DEFAULT))
+    def test_mid_volume_token_prices(self, environment_url, token):
+        self.run_coingecko_comparison_test(environment_url, token, MID_PRICE_DIFF)
+
+    # NUM_TOKENS_DEFAULT top by-volume tokens
+    @pytest.mark.parametrize("token", setup.choose_tokens_volume_range(NUM_TOKENS_DEFAULT))
+    def test_top_volume_token_prices(self, environment_url, token):
+        self.run_coingecko_comparison_test(environment_url, token, LOW_PRICE_DIFF)
+
+    def run_coingecko_comparison_test(self, environment_url, token, price_diff_threshold):
```

Comment:
Could we add a summary of what this helper does?

Similar to:
https://github.com/osmosis-labs/sqs/blob/3c66c5f7fe24c223fd73ff3cd9c0b1adefc69ff6/tests/setup.py#L259-L260

Reply:
Done